tests/lib/nested.sh

```diff
@@ -724,13 +742,13 @@ EOF
                     fi
 
                     # which channel?
-                    snap download --channel="$CORE_CHANNEL" --basename=core20 core20
-                    repack_core_snap_with_tweaks "core20.snap" "new-core20.snap"
-                    EXTRA_FUNDAMENTAL="$EXTRA_FUNDAMENTAL --snap $PWD/new-core20.snap"
+                    snap download --channel="$CORE_CHANNEL" --basename=core$VERSION core$VERSION
+                    repack_core_snap_with_tweaks "core${VERSION}.snap" "new-core${VERSION}.snap"
+                    EXTRA_FUNDAMENTAL="$EXTRA_FUNDAMENTAL --snap $PWD/new-core${VERSION}.snap"
 
                     # sign the snapd snap with fakestore if requested
                     if [ "$NESTED_SIGN_SNAPS_FAKESTORE" = "true" ]; then
-                        make_snap_installable_with_id --noack "$NESTED_FAKESTORE_BLOB_DIR" "$PWD/new-core20.snap" "DLqre5XGLbDqg9jPtiAhRRjDuPVa5X1q"
+                        make_snap_installable_with_id --noack "$NESTED_FAKESTORE_BLOB_DIR" "$PWD/new-core${VERSION}.snap" "DLqre5XGLbDqg9jPtiAhRRjDuPVa5X1q"
```

Comment:
this is wrong, you will need a if for core20 to use DLqre5XGLbDqg9jPtiAhRRjDuPVa5X1q and for core22 to use amcUKQILKXHHTlmSa7NMdnXSx02dNeeT

Reply:
done